src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/FluentBindingProviderExtensions.cs

```diff
@@ -11,16 +11,14 @@ namespace Microsoft.Azure.WebJobs.Host
 {    
     // Internal Extension methods for setting backwards compatibility hooks on certain bindings. 
     // This keeps the hooks out of the public surface. 
-    internal static class FluidBindingProviderExtensions
+    internal static class FluentBindingProviderExtensions
```

Comment:
there's still the logging hook to maintain dashboard compat. 

---
In reply to: [117080395](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117080395) [](ancestors = 117080395)